### ISSUE TITLE
Option: List instead of tiles

### ISSUE THREAD

**Issue by philiprhoades** [deweyapp/dewey#92]
I just found Dewey and it is quite nice but I find the tiles use too much screen real estate - is it possible to have an option for a list instead with the same info as the tiles have?

Thanks,

Phil.

**Comment by anshbansal** [comment]
Has this been added?

**Comment by jamiewilson** [comment]
@anshbansal No, I'm sorry, this hasn't been added yet. Still on the list though!

**Comment by hottehead** [comment]
+1 for this!

I know it's late but google still did not add tags to their system.
